service/qos/service_level_controller.hh

```diff
@@ -49,10 +49,18 @@ class service_level_controller : public peering_sharded_service<service_level_co
 public:
     class service_level_distributed_data_accessor {
     public:
+        virtual future<> init() = 0;
+        virtual future<> stop() = 0;
+
+        // virtual bool is_data_source_ready() const = 0;
         virtual future<qos::service_levels_info> get_service_levels() const = 0;
         virtual future<qos::service_levels_info> get_service_level(sstring service_level_name) const = 0;
         virtual future<> set_service_level(sstring service_level_name, qos::service_level_options slo) const = 0;
         virtual future<> drop_service_level(sstring service_level_name) const = 0;
+
```

Comment:
Did those commented out functions left here intentionally?
(I guess not and they should probably be cleaned up :slightly_smiling_face: )

Reply:
yep, it's a leftover I haven't noticed